Unable to find tinyxml2 in focal builds

Our nightly builds of `libsdformat10` are failing on focal due to problems finding tinyxml2:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat10-debbuilder&build=9)](https://build.osrfoundation.org/job/sdformat10-debbuilder/9/) https://build.osrfoundation.org/job/sdformat10-debbuilder/9/console

~~~
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find TinyXML2 (missing: TINYXML2_LIBRARY TINYXML2_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindTinyXML2.cmake:31 (find_package_handle_standard_args)
  cmake/SearchForStuff.cmake:11 (find_package)
  CMakeLists.txt:114 (include)


-- Configuring incomplete, errors occurred!
~~~

cc @mjcarroll

--- a/focal/debian/control
+++ b/focal/debian/control
@@ -6,7 +6,7 @@ Priority: optional
 Build-Depends: cmake,
                debhelper (>= 9),
                python3,
-               libtinyxml-dev,
+               libtinyxml2-dev,
                doxygen,
                texlive-base,
                texlive-fonts-recommended,
@@ -46,7 +46,7 @@ Description: Simulation Description Format (SDF) parser - Shared library
 Package: libsdformat10-dev
 Architecture: any
 Section: libdevel
-Depends: libtinyxml-dev,
+Depends: libtinyxml2-dev,
          libignition-tools-dev,
          libignition-math6-dev,
          libsdformat10 (= ${binary:Version}),

